data_formats-it.md

```diff
@@ -0,0 +1,66 @@
+# Formato dei dati
+
+
+## CSV
+
+- [Comma-separated values](https://it.wikipedia.org/wiki/Comma-separated_values) su Wikipedia.
+
+### Implementazioni
+
+- Python
+    - [csv](https://docs.python.org/3/library/csv.html) - una libreria integrata.
+- NodeJS
+    - [csv](https://csv.js.org/) - un pacchetto NPM.
+- PHP
+    - [fgetcsv](https://www.php.net/manual/en/function.fgetcsv.php) manuale di funzioni. Guarda anche [tutorial](https://phpenthusiast.com/blog/parse-csv-with-php) che ne fa uso.
+    - [str_getcsv](https://www.php.net/manual/en/function.str-getcsv.php) manuale di funzioni.
+- Shell
+    - [Bash Leggi file CSV separati da virgole su Linux / Unix](https://www.cyberciti.biz/faq/unix-linux-bash-read-comma-separated-cvsfile/) tutorial.
+    - comando `awk` 
+        - [Usare AWK per lavorare con i file CSV](https://newfivefour.com/awk-csv-files.html)
+        - [Usare il commando awk per i file .csv ](https://www.unix.com/shell-programming-and-scripting/124886-using-awk-command-csv-file.html)
+    - commando `gawk` 
+        - [leggere file](https://www.gnu.org/software/gawk/manual/html_node/Reading-Files.html) sulla documentazione di GNU.
+    - commando `sed` 
+        - [sed - 10 esempi per sostituire / cancellare / stampare linee di file CSV ](https://www.theunixschool.com/2013/02/sed-examples-replace-delete-print-lines-csv-files.html)
+        - [FAQ sed - Come posso analizzare un file di dati delimitato da virgole (CSV)?](https://www.linuxtopia.org/online_books/linux_tool_guides/the_sed_faq/sedfaq4_005.html)
+
+
+## JSON
+
+- [json.org](https://www.json.org)
+- [JSON](https://www.w3schools.com/js/js_json_intro.asp) su W3 Schools.
+
+### Varianti di JSON
+
+- [Jsonnet](https://jsonnet.org/)
+- [JSONP](https://www.w3schools.com/js/js_json_jsonp.asp)
+
+### Implementazioni
+
+- Python
+    - [json]((https://docs.python.org/3/library/json.html)
```

Comment:
```suggestion
    - [json](https://docs.python.org/3/library/json.html)
```

Reply:
Whoops, sorry, I'll fix it now

Reply:
The bug was in the English one so I fixed that now. Oops.